containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
> Error establishing a database connection

Sounds like the wrong environment might be loaded? I would check the output of `docker inspect` to see which environment variables the container is using. (this may not be helpful, as the problem could be an incorrect value in `${WORDPRESS_BUILD_DIR}` which is used when building the WordPress container)